erica/application/tax_declaration/tax_declaration_dto.py

```diff
@@ -1,7 +1,10 @@
+from pydantic.main import BaseModel
+
 from erica.application.base_dto import BaseDto
+from erica.domain.TaxDeclaration.TaxDeclaration import TaxDeclarationPayload
 from erica.erica_legacy.request_processing.erica_input.v1.erica_input import FormDataEst, MetaDataEst
 
 
-class TaxDeclarationDto(BaseDto):
-    est_data: FormDataEst
-    meta_data: MetaDataEst
+class TaxDeclarationDto(BaseModel):
```

Comment:
Shouldn't we then move the dtos to the api folder? They are only supposed to be used in the api right?

Reply:
I thought so, too. But there might be something we are missing. Wdyt, @punknoir101 ?

Reply:
Thank you for explaining. There was something that we were missing then 😄

In my head the DTOs would only be used in the api, but if for example we would have a CLI, I could also have the same dto as arguments to the command-line, correct?

Reply:
Right.